Bottom: 26c065206f7e34ef10ad46ce1654a841a1bf34d9
Top:    89e3e265c4ea9adfdf0884d87fe0da1e86b27c12
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-02 16:14:22 +0800

Refresh of autofs-5.0.7-use-MS_UNBINDABLE-for-autofs-mounts.patch

---

diff --git a/daemon/spawn.c b/daemon/spawn.c
index 0f7d56a..3b4a009 100644
--- a/daemon/spawn.c
+++ b/daemon/spawn.c
@@ -24,6 +24,7 @@
 #include <poll.h>
 #include <sys/wait.h>
 #include <sys/stat.h>
+#include <sys/mount.h>
 
 #include "automount.h"
